src/main/diagnostics/abstractCodeActionProvider.ts

```diff
@@ -3,13 +3,13 @@ import { ExtensionComponent } from '../extensionComponent';
 import { DependenciesTreeNode } from '../treeDataProviders/dependenciesTree/dependenciesTreeNode';
 import { TreesManager } from '../treeDataProviders/treesManager';
 import { Severity, SeverityUtils } from '../types/severity';
-import { DiagnosticsUtils } from './diagnosticsUtils';
 
 /**
  * @see DiagnosticsManager
  */
 export abstract class AbstractCodeActionProvider implements vscode.CodeActionProvider, ExtensionComponent {
```

Comment:
```suggestion
export abstract class AbstractCodeActionProvider implements vscode.CodeActionProvider, ExtensionComponent, vscode.Disposable {
```

Please also push "this" to the context subscriptions:
```go
context.subscriptions.push(
    this,
    vscode.languages.registerCodeActionsProvider(this._documentSelector, this, {
        providedCodeActionKinds: [vscode.CodeActionKind.Empty]
    }),
);
```